lib/ansible/errors/__init__.py

```diff
@@ -19,11 +19,18 @@
 from __future__ import (absolute_import, division, print_function)
 __metaclass__ = type
 
-import os
-
 from ansible.errors.yaml_strings import *
 from ansible.utils.unicode import to_unicode, to_bytes
 
+
+if str is bytes:
+    # Python 2
+    to_str = to_bytes
+else:
+    # Python 3
+    to_str = to_unicode
+
```

Comment:
We should probably put this into ansible.utils.unicode with a note about when to use it and what sort of literals to use it with for those that don't understand the native string distinction.